airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -349,6 +349,13 @@
   documentationUrl: https://docs.airbyte.com/integrations/sources/faker
   sourceType: api
   releaseStage: alpha
+- name: Faker
```

Comment:
Please correct the name of the connector.

Reply:
Done